scripts/azureml-assets/azureml/assets/release/model_publish_utils/utils.py

```diff
@@ -49,33 +42,64 @@ def _run(self, cmd, cwd: Path = "./") -> int:
             logger.print(f"Successfully executed! Output: \n{result.stdout}")
         return result.returncode
 
-    def _download_git_model(self) -> bool:
-        """Download the Model."""
-        """
-        Clones the model from the git URL into the Model Directory.
-        Deletes the incomplete model artifact in case of failure.
-        Sets the HEAD to the commit hash.
-        Deletes the .git folder in the downloaded artifacts.
+    def _download_git_model(model_uri: str, model_dir: Path) -> bool:
+        """Download model files from GIT repository.
+
+        :param model_url: git clonable uri of a public repo
+        :type model_url: str
+        :param model_dir: local directory to clone model to
+        :type: Path
         """
-        clone_cmd = f"git clone {self.model_url} {self.model_dir}"
-        result = self._run(clone_cmd)
+        clone_cmd = f"git clone {model_uri} {model_dir}"
+        result = ModelDownloadUtils._run(clone_cmd)
         if result != 0:
             return False
-        if self.model_commit_hash:
-            cmd = f"git reset --hard {self.model_commit_hash}"
-            commit_exists = self._run(cmd, cwd=self.model_dir)
-            if commit_exists != 0:
-                # TODO: Error handling incase of incorrect commit hash.
-                logger.log_warning("Commit hash doesn't exist. Using model from latest HEAD.")
-        git_path = os.path.join(self.model_dir, '.git')
+        git_path = model_dir / ".git"
         shutil.rmtree(git_path, onerror=_onerror)
         return True
 
-    def download_model(self) -> bool:
-        """Prepare the Download Environment."""
-        if self.model_download_type == ModelDownloadType.GIT:
-            download_success = self._download_git_model()
-            return download_success
+    def _download_azure_artifacts(model_uri, model_dir) -> bool:
+        """Download model files from blobstore.
+
+        :param model_url: Publicly readable blobstore URI of model files
+        :type model_url: str
+        :param model_dir: local directory to download model to
+        :type: Path
+        """
+        try:
+            SP_CLIENT_ID = os.environ["SP_CLIENT_ID"]
+            SP_TENANT_ID = os.environ["SP_TENANT_ID"]
+            login = f"azcopy login --service-principal --application-id {SP_CLIENT_ID} --tenant-id {SP_TENANT_ID}"
+            result = ModelDownloadUtils._run(login)
+            if result:
+                logger.log_error("azcopy failed to login")
+                return False
+            download_cmd = f"azcopy cp --recursive=true {model_uri} {model_dir}"
```

Comment:
Could you please update this command to output verbose logs?

Reply:
`azcopy` has default log verbosity set to INFO.

![image](https://user-images.githubusercontent.com/61145377/204712839-9690fd31-01ca-4778-84c4-1658bdc43bb3.png)